pkg/plugins/common/kustomize/v2-alpha/scaffolds/internal/templates/config/manager/config.go

```diff
@@ -105,6 +105,8 @@ spec:
       #               - linux
       securityContext:
         runAsNonRoot: true
+        # if your K8s version <= 1.21, may need to uncomment the follow fsGroup to get SA token file access.
```

Comment:
Hi @astraw99,

I do not think that we should add/change the scaffold.
(IHMO)

a) It could be fit in an issue as a workaround (so that others can find the problem and solution)
b) We could add it in the docs under a new FAQ section  
c) It would be great better understand it and make clear that ONLY happens in development env when we are running/testing the project OUTSIDE of the cluster with make run.

Reply:
/hold

Reply:
OK, maybe it is more proper to add this case into the FAQ section (I am doing).